File: patterns/c-patterns/pattern74.c
// Pattern 74: ...

// * * * * * * *
// *         *
// *       *
// *     *
// *   *
// * *
// *

#include <stdio.h>

int main()
{
    int n = 7;
    int i, j;

    for (i = n; i >= 1; i--)
    {
        for (j = n; j >= 1; j--)
        {
            if (j == n - i + 1 || j == n || i == n)
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern74.py
# Pattern 74: ...

# * * * * * * *
# *         *
# *       *
# *     *
# *   *
# * *
# *

n = 7
for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        if j == n - i + 1 or j == n or i == n:
            print("* ", end="")
        else:
            print("  ", end="")
    print()
